src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -5,10 +5,13 @@
 from pathlib import Path
```

Comment:
I assume we don't have any tests for the tools yet right?

Reply:
no we don't. No tests for any tool computation codes.

Reply:
Fair enough :+1: